### DISCUSSION TITLE
Upload attachments cloundfont error 405

### DISCUSSION THREAD

**Discussion by hoanhlv050588** [indrawp/Setup-Guide#54] [Q&A]
Hi.

I create a bucket, then create a folder called “attachments /”.
![addtach](https://user-images.githubusercontent.com/48352319/113466907-aa9fac00-9469-11eb-9e44-d59c8004adaf.PNG)

When I upload the file, the error return code is 405
![addtach2](https://user-images.githubusercontent.com/48352319/113466913-b1c6ba00-9469-11eb-913c-223256cfd7f7.PNG)

If me ignore the attachments folder and directly upload it to the root directory, it will be successful.
How do I configure it? to be able to upload to the “attachments” folder.

Please help me. Thank !

**Comment by madeindra** [comment]
@hoanhlv050588 This is a common problem I found since AWS have a renewed design months ago, 405 means that method, which is POST, is not allowed, could be a missing permission

For now, you can try to use MinIO.

**Reply by hoanhlv050588** [reply]
thank U!
I will use MinIO.
